### PR TITLE
Tweaks, mixed bag

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by msrb** [comment]
I have no idea if this works or not... therefore WIP status.

**Comment by tisnik** [comment]
@msrb I know this is in WIP state, but:
1. the unit tests are disabled by this PR. Is it on purpose?
2. I'm not sure how the /liveness and /readiness endpoints works right now. Don't they point to / (which might not be related much to the state of the whole service)?

**Comment by msrb** [comment]
@tisnik 

1. Yes, tests are disabled on purpose. There are barely any tests right now and thus I don't consider them being a good indicator whether the service is really working or not. Test suite will need to be extended. The purpose of this PR is to fix what's fundamentally wrong with current implementation.

2. What do you mean? Having separate endpoints called `/liveness` and `/readiness` seems superfluous to me. Especially if those endpoints only do `return jsonify({}), 200`, nothing else. Those endpoints are not useful for users, they only make the API more complicated.

**Comment by tisnik** [comment]
@msrb 

1. I agree. Do we have any plans (an issue) for improving this part? I mean both unit test and integration tests as well (those are more important for such type of service, at least IMHO :)

2. yup, but they still exists and points to /, right? I.e. we got the response that the service is ready even if any config is wrong (list of users for example).

File: gateway/api_v1.py [hunk 1]
```diff
@@ -40,70 +41,50 @@ def index():
     return jsonify(response), 200
 
 
-@app.route('/<service>/<path:varargs>', methods=['POST', 'GET'])
+@app.route(
+    '/<service>/<path:varargs>',
+    methods=['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'CONNECT', 'OPTIONS', 'TRACE', 'PATCH']
```
Comment (humaton) [inline gateway/api_v1.py:46]: Do we use all of the listed methods in our services?

File: gateway/api_v1.py [hunk 2]
```diff
@@ -40,70 +41,50 @@ def index():
...
+)
+@app.route(
+    '/<service>/',
+    methods=['GET', 'HEAD', 'POST', 'PUT', 'DELETE', 'CONNECT', 'OPTIONS', 'TRACE', 'PATCH'],
```
Comment (humaton) [inline gateway/api_v1.py:50]: Same as above.

**Review by humaton — APPROVED** [review]
LGTM, just one question inline

File: gateway/api_v1.py (hunk 1 above)
Comment (msrb) [reply gateway/api_v1.py:46]: We don't. But API gateway should not make any assumptions here, IMO. It simply passes requests to underlying services and it's up to the services to return 405 if given method is not supported.

**Pull request merged**